starter-code/models/shop.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema   = mongoose.Schema;
+
+
+const shopSchema = new Schema({
+  name: { type: String },
+  description: { type: String },
+  location: {
+    type: { type: String },
+    coordinates: [Number]
+  }
```

Comment:
You can also include a field to if there are a list of possible values, you can use enum:

```javascript
category: {
  type: String,
  enum : ['coffe','bookstore'],
}
```